package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -180,6 +180,11 @@ If you use the Mean Squared Displacement analysis code in
 
 .. _`10.21105/joss.00877`: https://doi.org/10.21105/joss.00877
 
+.. [Corradi2018] Corradi, V. et al. Lipid–Protein Interactions Are Unique Fingerprints for Membrane Proteins. ACS Cent Sci 4, 709–717 (2018). doi:`10.1021/acscentsci.8b00143`_
```

Comment:
Please include all authors... bytes aren't that expensive nowadays, we can afford all authors.